backpack/extensions/backprop_extension.py

```diff
@@ -74,38 +89,56 @@ def set_module_extension(
             )
         self.__module_extensions[module] = extension
 
-    def __get_module_extension(self, module):
+    def __get_module_extension(
+        self, module: Module
+    ) -> Callable[[BackpropExtension, Module, Tuple[Tensor], Tuple[Tensor]], None]:
         module_extension = self.__module_extensions.get(module.__class__)
 
         if module_extension is None:
-
-            if isinstance(module, (Sequential, ReduceTuple)):
-                return no_op
-
             if self.__fail_mode is FAIL_ERROR:
                 raise NotImplementedError(
-                    "Extension saving to {} ".format(self.savefield)
-                    + "does not have an extension for "
-                    + "Module {}".format(module.__class__)
+                    f"Extension saving to {self.savefield} "
+                    "does not have an extension for "
+                    f"Module {module.__class__}"
                 )
             elif self.__fail_mode == FAIL_WARN:
-                warnings.warn(
-                    "Extension saving to {} ".format(self.savefield)
-                    + "does not have an extension for "
-                    + "Module {}".format(module.__class__)
-                )
+                for _ in module.parameters():
+                    warnings.warn(
+                        f"Extension saving to {self.savefield} does not have an "
+                        f"extension for Module {module.__class__} "
+                        f"although the module has parameters"
+                    )
+                    break
 
-            return no_op
+            return no_op_apply
 
         return module_extension.apply
 
-    def apply(self, module, g_inp, g_out):
+    def apply(
```

Comment:
Let's use python's built-in `__call__` method to make an object callable: Rename into `__call__`. Replace `backprop_extension.apply(...)` by `backprop_extension(...)` in  `__init__.py`.

Reply:
Nice idea. Implemented it.